### DISCUSSION TITLE
Nested error

### DISCUSSION THREAD

**Discussion by niralparmar** [react-hook-form/react-hook-form#3621] [Q&A]
hi,
i am trying to display error for my nested input, "errors" object returns "undefined" if i try to access through nested field name because errors object is nested.

here is my codesandbox : [https://codesandbox.io/s/react-hook-form-nested-error-13gzp?file=/src/index.js](url)

how can i fix this.

i appreciate any help

**Comment by bluebill1049** [comment]
use optional chain or lodash `get`

**Reply by niralparmar** [reply]
option chain wont work in my case as i have common input component and it could have any level of nested field.
